src/lib.rs

```diff
@@ -47,16 +47,24 @@ macro_rules! register_module {
     ($module:pat, $init:block) => {
         #[no_mangle]
         pub unsafe extern "C" fn napi_register_module_v1(
-            _env: $crate::macro_internal::runtime::nodejs_sys::napi_env,
-            exports: $crate::macro_internal::runtime::nodejs_sys::napi_value
+            env: $crate::macro_internal::runtime::nodejs_sys::napi_env,
+            m: $crate::macro_internal::runtime::nodejs_sys::napi_value
         ) -> $crate::macro_internal::runtime::nodejs_sys::napi_value
         {
             // Suppress the default Rust panic hook, which prints diagnostics to stderr.
-            ::std::panic::set_hook(::std::boxed::Box::new(|_| { }));
+            // FIXME: Need this for debugging unimplemented panics
+            // ::std::panic::set_hook(::std::boxed::Box::new(|_| { }));
```

Comment:
I wasn't seeing the backtraces, even with `RUST_BACKTRACE=1` when the panic hook was enabled. This made finding the method that was `unimplemented!()` difficult.

Do you have a suggestion for this other than commenting it out?

Reply:
If we want it to be a dynamic decision we can create a Neon-specific environment variable (maybe `NEON_VERBOSE` or `NEON_DEV` or something) for debugging.

Or maybe what we should actually do is gather the backtrace metadata programmatically and when we convert the panic to a JS exception, serialize the metadata as JS values and attach it to the JS error object. But we could do this in a separate PR.

The reason I disabled the hook was that since we convert Rust panics to JS exceptions, we don’t want to pollute end users’ experiences with a bunch of low level Rust spew. For example, a CLI tool that happens to have a deeply nested dependency on a Neon module shouldn’t spit out Rust metadata when some internal failure occurs.

Reply:
What do you think about reusing the `RUST_BACKTRACE` environment variable instead of introducing a new one?